packages/server/lib/modes/run.js

```diff
@@ -824,6 +824,10 @@ module.exports = {
       return
     }
 
+    if (quiet === true) {
```

Comment:
since we know it's a boolean at this point:

```suggestion
    if (quiet) {
```